tests/test_taskgroups.py

```diff
@@ -165,9 +165,11 @@ async def taskfunc(*, task_status):
         finished = True
 
     async def start_another():
+        nonlocal tg
```

Comment:
Why have you added this?

Reply:
I think I added some assertions to the test before removing them. I've removed this line